packages/framework/src/Controller/Admin/MailController.php

```diff
@@ -183,4 +194,56 @@ public function settingAction(Request $request): Response
             'form' => $form->createView(),
         ]);
     }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @param int $mailTemplateId
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    #[Route(path: '/mail/send/{mailTemplateId}', requirements: ['mailTemplateId' => '\d+'])]
```

Comment:
Shouldn't the route only be accessible for XHR?

now I can open the route in the browser 
![Snímek obrazovky 2024-12-10 v 21 14 06](https://github.com/user-attachments/assets/484b9ce9-9101-48d9-9653-37611aff8063)

Reply:
I had in mind a scenario (for a developer) when you need to quickly test something multiple times, it could ease the task for you but I do not need that :slightly_smiling_face: 7baa7349eb